### PR TITLE
converted string variables to woql variables Var

### PR DESCRIPTION
**Pull request by Neelterminusdb**
closes https://github.com/terminusdb/terminusdb-docs/issues/111

Signed-off-by: NeelParihar <neel@terminusdb.com>

### CODE REVIEW

**Review by spl — CHANGES_REQUESTED** [review]
Just some minor changes needed. I realized I left my comments on the `.md` when they should've been on the `.js`, so just keep that in mind.

File: docs/api/woql.js.md [hunk 1]
```diff
@@ -862,7 +878,8 @@ takes a variable number of string arguments and concatenates them into a single
 
 **Example**  
 ```javascript
-concat(["v:first_name", " ", "v:last_name"], "v:full_name")
+let [first_name, last_name] = vars("first_name", "last_name")
+concat([first_name, " ", last_name], full_name)
 WOQL.concat(["first_name", " ", "last_name"], "full_name")
```
[the remark this answers is not in the document]
Comment (Neelterminusdb) [reply docs/api/woql.js.md:883]: Yup forgot about that, will change it

**Review by spl — CHANGES_REQUESTED** [review]
In addition to these, it looks like the previous issues I mentioned have not been resolved: indentation, semicolons, and `"full_value"`.

File: docs/api/woql.js.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (spl) [reply docs/api/woql.js.md:883]: The `concat` seems to be duplicated here:

```
concat([first_name, " ", last_name], full_name)
WOQL.concat(["first_name", " ", "last_name"], "full_name")
```

**Review by spl — APPROVED** [review]

**Pull request merged**